packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -273,6 +273,7 @@ const start = async (zcf, privateArgs) => {
   );
 
   const creatorFacet = wrapCreatorFacet(
+    // @ts-expect-error ??? is there any constraint on what can be wrapped? Reading elsewhere I thought getGovernedApis() was required
```

Comment:
My best approach at the moment is to require it, and in contracts that don't have any governed APIs (no methods that should be callable under governance) `getGovernedApis()` returns `empty`or throws.  Is there a better way to implement this or a better way to declare it?

Reply:
Let's hash out the design https://docs.google.com/document/d/1tePGqiu-c0wXjIdR9_IWqWP9UKIgHQeoujg7eW7lh70/edit?usp=sharing